docs/services/rabbitmq/upgrade.md

```diff
@@ -0,0 +1,37 @@
+# Upgrading RabbitMQ 
```

Comment:
we should also create a file in parent https://github.com/dimagi/commcare-cloud/blob/master/docs/services and link this file there.